src/lib/UI/Module/FieldTypeToolbar/FieldTypeToolbarFactory.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\UI\Module\FieldTypeToolbar;
+
+use eZ\Publish\Core\FieldType\FieldTypeRegistry;
+use EzSystems\EzPlatformAdminUi\UI\Module\FieldTypeToolbar\Values\FieldTypeToolbar;
+use EzSystems\EzPlatformAdminUi\UI\Module\FieldTypeToolbar\Values\FieldTypeToolbarItem;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class FieldTypeToolbarFactory
```

Comment:
is it internal?

Reply:
It seems it's only used for `ContentTypeController`, to provide parameters to view.

So I guess I'll mark it as `@internal` unless otherwise. @adamwojs might want to answer this once he's back. Maybe... :sweat_smile: 

Fixed in 1959048b.